src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/carts.yml

```diff
@@ -59,6 +59,28 @@ admin_carts_edit_carrier:
   options:
     expose: true
 
+admin_carts_edit_currency:
+  path: /{cartId}/currency
+  methods: [POST]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Order/Cart:editCurrency
+    _legacy_controller: AdminCarts
```

Comment:
Could you add the `_legacy_link` here please? Even if they are commented for now

Reply:
As mentioned in gitter, this route isnt matching legacy behavior so it doesn't need legacy link. However i added missing legacy link in admin_carts_view route